### PR TITLE
TASK-39555: Tasks not displayed after deletion without refreshing the page

### PR DESCRIPTION
**Pull request by NaderSai**
Task not displayed after deletion and without refreshing the page!

### CODE REVIEW

File: task-management/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,13664 @@
+{
```
Comment (ahamdi) [inline task-management/package-lock.json]: Do not commit this file , please remove it from the PR

File: task-management/pom.xml [hunk 2]
```diff
@@ -32,7 +32,7 @@
   <properties>
     <!-- Maven Plugin for JS and CSS bundling and Unit Test execution -->
     <com.github.eirslett.frontend.version>1.6</com.github.eirslett.frontend.version>
-    <node.version>v11.4.0</node.version>
+    <node.version>v12.0.0</node.version>
```
Comment (ahamdi) [inline task-management/pom.xml]: We do not change this on maintenance, please restore default verion of Node

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: task-management/package-lock.json (hunk 1 above)
Comment (NaderSai) [reply task-management/package-lock.json]: done

File: task-management/pom.xml (hunk 2 above)
Comment (NaderSai) [reply task-management/pom.xml]: done

**Review by aelbenney — APPROVED** [review]

**Review by ahamdi — APPROVED** [review]

**Pull request merged**